{"tests": [{"data": {"status": "error", "tool_id": "IDconverter", "time_seconds": 0.0345609188079834, "inputs": {"species|idtypein": "UniProt-AC", "input|ncol": "c1", "input|header": true, "species|mapping_file": "human_id_mapping", "input|file": {"src": "hda", "id": "2891970512fa2d5a"}, "species|idto|idtypeout": ["neXtProt", "UniProt-ID", "GeneID", "MIM", "Ensembl"], "input|ids": "file"}, "tool_version": "2019.06.18", "problem_log": "Traceback (most recent call last):\n  File \"/usr/lib/python2.7/unittest/case.py\", line 329, in run\n    testMethod()\n  File \"/tmp/tmpPDaobo/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmpPDaobo/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmpPDaobo/galaxy-dev/lib/galaxy/tools/verify/interactor.py\", line 764, in verify_tool\n    raise e\nRunToolException: Error creating a job for these tool inputs - An invalid option was selected for idtypeout, u'Ensembl', please verify., Parameter mapping_file requires a value, but has no legal values defined.\n", "test_index": 0, "problem_type": "galaxy.tools.verify.interactor.RunToolException", "execution_problem": "Error creating a job for these tool inputs - An invalid option was selected for idtypeout, u'Ensembl', please verify., Parameter mapping_file requires a value, but has no legal values defined."}, "id": "functional.test_toolbox.TestForTool_IDconverter.test_tool_000000", "has_data": true}], "version": "0.1", "exit_code": 1, "summary": {"num_skips": 0, "num_errors": 1, "num_failures": 0, "num_tests": 1}}